src/config/express.js

```diff
@@ -1,71 +1,85 @@
 const express = require('express')
 const router = express.Router()
 const path = require('path')
-const env = require('./env')
 const favicon = require('serve-favicon')
 const bodyParser = require('body-parser')
-const session = require('express-session')
-const cookieParser = require('cookie-parser')
 const BunyanMiddleware = require('bunyan-middleware')
 const HttpStatus = require('http-status-codes')
 
 module.exports = logger => {
-    /**
-     * Global App Config
-     */
+  /**
+   * Global App Config
+   */
   let app = express()
-
   app.use(BunyanMiddleware({
     headerName: 'X-Request-Id',
     propertyName: 'reqId',
     logName: 'req_id',
     obscureHeaders: [],
     logger: logger
   }))
+  app.use(bodyParser.json())
+  app.use(bodyParser.urlencoded({
+    extended: true
+  }))
+
+  app.use(favicon('./public/img/favicon.ico'))
+  app.use(express.static(path.join(__dirname, 'public')))
 
-  app.use(session({
-    saveUninitialized: true,
-    resave: true,
-    secret: env('SESSION_SECRET')
+  app.use(BunyanMiddleware({
+    headerName: 'X-Request-Id',
+    propertyName: 'reqId',
+    logName: 'req_id',
+    obscureHeaders: [],
+    logger: logger
   }))
 
   app.use(bodyParser.json())
-
   app.use(bodyParser.urlencoded({
     extended: true
   }))
 
-  app.use(cookieParser())
-
   app.use(favicon('./public/img/favicon.ico'))
-
   app.use(express.static(path.join(__dirname, 'public')))
 
-    /**
-     * Routing Config
-     */
+  /**
+   * Routing Config
+   */
   app.use('/api', router)
 
+  require('../routes/auth.routes')(router)
+  require('../routes/user.routes')(router)
```

Comment:
The router config gets injected to every route when they get initialized. A best practice to add a prefix on every route.